code/game/objects/items/crayons.dm

```diff
@@ -428,11 +428,11 @@
 		target = target.loc
 
 	if(!isturf(target))
-		return
+		return FALSE
```

Comment:
The documentation for this proc says it `Returns how many charges were used.` so we might want to update that

...But blech, I don't really like resorting to this kind of return values, it's just confusing and can easily be screwed up by someone who doesn't understand what's going on. 

Can we have a separate proc `can_use_on` (which is effectly just, `if(isturf(target))`, that runs before `use_on`? Then that can return true to run `use_on` (and always return blocking) or `false` to not block